common/autoware_universe_utils/include/autoware/universe_utils/geometry/geometry.hpp

```diff
@@ -572,17 +572,70 @@ inline double calcNorm(const geometry_msgs::msg::Vector3 & v)
 //
 bool isTwistCovarianceValid(const geometry_msgs::msg::TwistWithCovariance & twist_with_covariance);
 
-// NOTE: much faster than boost::geometry::intersects()
-std::optional<geometry_msgs::msg::Point> intersect(
-  const geometry_msgs::msg::Point & p1, const geometry_msgs::msg::Point & p2,
-  const geometry_msgs::msg::Point & p3, const geometry_msgs::msg::Point & p4);
-
 /**
  * @brief Check if 2 convex polygons intersect using the GJK algorithm
  * @details much faster than boost::geometry::intersects()
  */
 bool intersects_convex(const Polygon2d & convex_polygon1, const Polygon2d & convex_polygon2);
 
+// Alternatives for Boost.Geometry ----------------------------------------------------------------
+
+// TODO(mitukou1109): remove namespace when migrating to alternatives
+namespace alt
+{
+using Point = tf2::Vector3;
+using CvxPolygon = std::vector<tf2::Vector3>;
+using PointList = std::vector<tf2::Vector3>;
+
+alt::Point fromGeom(const geometry_msgs::msg::Point & point);
```

Comment:
According to the coding guideline, please use the snake_case instead of camelCase.